packages/manager/modules/pci/src/projects/project/instances/start/start.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.instances.start', {
+    url: '/start?instanceId',
+    views: {
+      modal: {
+        component: 'pciInstancesInstanceStart',
```

Comment:
Same remark as `ovhManagerPciInstancesStop` module.

Both `@ovh-ux/ng-ui-router-layout` and `@ovh-ux/ng-ui-router-breadcrumb` aren't supposed to be defined as a dependency?

Reply:
Deps. has been added